### PR TITLE
New package: catch2-2.7.0.

### PR DESCRIPTION
**Pull request by DBLouis**

### CODE REVIEW

File: srcpkgs/catch2/template [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+hostmakedepends="python3"
+short_desc="C++ header-only test framework for unit-tests, TDD and BDD"
+maintainer="Louis Dupré Bertoni <contact@louisdb.xyz>"
+license="BSL-1.0"
+homepage="https://github.com/catchorg/Catch2"
+distfiles="https://github.com/catchorg/Catch2/archive/v${version}/${pkgname}-v${version}.tar.gz"
+checksum=d4655e87c0ccda5a2e78bf4256fce8036feb969399503dcc8272f4c90347d9c0
+
+post_install() {
+	vlicense "${wrksrc}/LICENSE.txt"
```
Comment (maxice8) [inline srcpkgs/catch2/template]: `vlicense LICENSE.txt`

File: srcpkgs/catch2/template [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# Template file for 'catch2'
+pkgname=catch2
+version=2.7.0
+revision=1
+wrksrc="Catch2-${version}"
+build_style=cmake
+configure_args="
+	-DCATCH_USE_VALGRIND=OFF
```
Comment (maxice8) [inline srcpkgs/catch2/template]: one whitespace to indent, keep multiple options in a line and cut at 80 columns

File: srcpkgs/catch2/template [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+	-DCATCH_BUILD_TESTING=ON
+	-DCATCH_BUILD_EXAMPLES=OFF
+	-DCATCH_BUILD_EXTRA_TESTS=OFF
+	-DCATCH_ENABLE_COVERAGE=OFF
+	-DCATCH_ENABLE_WERROR=ON
```
Comment (maxice8) [inline srcpkgs/catch2/template]: set it to OFF

File: srcpkgs/catch2/template [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+homepage="https://github.com/catchorg/Catch2"
+distfiles="https://github.com/catchorg/Catch2/archive/v${version}/${pkgname}-v${version}.tar.gz"
+checksum=d4655e87c0ccda5a2e78bf4256fce8036feb969399503dcc8272f4c90347d9c0
+
+post_install() {
+	vlicense "${wrksrc}/LICENSE.txt"
+}
+
+do_check() {
+	cd "${wrksrc}/build"
```
Comment (maxice8) [inline srcpkgs/catch2/template]: cd build

File: srcpkgs/catch2/template (hunk 2 above)
Comment (maxice8) [reply srcpkgs/catch2/template]: .

**Comment by DBLouis** [comment]
Why did you close it?

**Comment by maxice8** [comment]
On Wed, Mar 20, 2019 at 11:03:02AM -0700, LouisDB wrote:
> Why did you close it?
> 
> -- 
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/10087#issuecomment-474961477

Merged

**Comment by DBLouis** [comment]
> On Wed, Mar 20, 2019 at 11:03:02AM -0700, LouisDB wrote: Why did you close it? -- You are receiving this because you modified the open/close state. Reply to this email directly or view it on GitHub: [#10087 (comment)](https://github.com/void-linux/void-packages/pull/10087#issuecomment-474961477)
> Merged

Sorry, I did not see

**Comment by maxice8** [comment]
On Wed, Mar 20, 2019 at 11:07:14AM -0700, LouisDB wrote:
> > On Wed, Mar 20, 2019 at 11:03:02AM -0700, LouisDB wrote: Why did you close it? -- You are receiving this because you modified the open/close state. Reply to this email directly or view it on GitHub: [#10087 (comment)](https://github.com/void-linux/void-packages/pull/10087#issuecomment-474961477)
> > Merged
> 
> Sorry, I did not see
> 
> -- 
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/10087#issuecomment-474963425

No worries, it appears closed because i pushed it manually instead of using the WebUI merge button or the github API for merging.

**Pull request closed without merging**